utils/installer/install.sh

```diff
@@ -63,6 +69,10 @@ installpiponubuntu() {
 	sudo apt install python3-pip >/dev/null
 }
 
+installpipontermux() {
+	apt install python
```

Comment:
will this install `pip` as well?

Reply:
Termux doesn't come with python3 by default to install it we use `apt install python` and it installs `pip`, make, clang ...
btw i just retested it and it works